src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -33,6 +33,12 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(UseDotNetNativeToolchain)' == 'true' And '$(TestAppX)' != 'true'">
+    <TestILCFolder>%PACKAGE_DIR%\TestILC\1.0.0</TestILCFolder>
```

Comment:
This isn't really a nupkg yet... need to finalize how to package the toolchain.  Currently this zip is on a share on my personal box.